src/scripts/main.js

```diff
@@ -2,4 +2,22 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  e.preventDefault();
```

Comment:
Calling `e.preventDefault()` without checking the event target might not be necessary here. If the default action is not needed to be prevented for all clicks, consider moving this line inside the condition that checks if the click is within the wall.